### ISSUE TITLE
Scroll to top problem

### ISSUE THREAD

**Issue by SubZeroX** [chetachiezikeuzor/Yin-and-Yang-Theme#51]
I don't really know why, but on long notes everytime I switch back and forth edit/preview, the document scroll to top. Must be related to the "Relationship lines" because when I removed this portion, it works just fine...

https://user-images.githubusercontent.com/13406988/133132264-e6ee6d18-914f-47a3-9f55-7deb90ea6342.mp4

**Comment by chetachiezikeuzor** [comment]
Yes this is the relationship lines. There is a setting to remove them using Style Settings. If this is how you removed them then you should be fine, I’ll try and add an alternative relationship line setting as well.
